.github/workflows/on-push.yaml

```diff
@@ -19,7 +19,6 @@ jobs:
           toolchain: stable
       - uses: taiki-e/install-action@cargo-hack
       - run: cargo hack check --each-feature
-      - run: RUSTFLAGS="--cfg loom" cargo hack check --each-feature
```

Comment:
This was never supposed to have been present, so I took it out. It's left over from another repo I made a few months ago where I made this template originally.